docs-public/snippets.js

```diff
@@ -199,14 +199,36 @@ function reactPage(html, jsx) {
     });
 }
 
+var groupExtractor = /\((.*?)\)/g;
+function extractRules(value) {
```

Comment:
This eats every parenthesis in the value, while the intent seems to be to trim only the first / last parenthesis. Maybe `/^\((.*?)\)$/` yields better results?